test/integration/mapping_validation_test.rb

```diff
@@ -111,6 +111,37 @@ def setup
     end
   end
 
+  test "Shopify taxonomy version is in consistent between VERSION file and mappings in the /data folder" do
+    shopify_taxonomy_version_from_file = "shopify/" + @sys.read_file("VERSION").strip
```

Comment:
_nit_ Not a blocker for the PR, but it would be nice to have a more standard way of getting the "currently active version"